suite-native/message-system/src/components/KillswitchMessageScreen.tsx

```diff
@@ -32,31 +34,24 @@ const buttonsWrapperStyle = prepareNativeStyle(_ => ({
     width: '100%',
 }));
 
-export const FeatureMessageScreen = () => {
+export const KillswitchMessageScreen = () => {
     const dispatch = useDispatch();
     const openLink = useOpenLink();
-
-    const activeFeatureMessages = useSelector(selectActiveFeatureMessages);
-    const firstFeatureMessage = A.head(activeFeatureMessages);
-
     const { applyStyle } = useNativeStyles();
 
-    if (!firstFeatureMessage) return null;
+    const killswitch = A.head(useSelector(selectActiveKillswitchMessages));
```

Comment:
nit: What about changing this selector to `selectFirstActiveKillswitchMessage` and put the `A.head` inside?

Reply:
49a065c965

Reply:
Even `selectActiveKillswitchMessage` (without the `First`) sounds good to me for this (only one can be active...)